qutip/floquet.py

```diff
@@ -117,13 +112,7 @@ def floquet_modes(H, T, args=None, sort=False, U=None):
     else:
```

Comment:
Not sure this is intended to sort according to `-e_quasi`. I would have expected the sort to be made according to `e_quasi` so that energies are in increasing order. But I should have missed something :/